src/components/molecules/PageFilter/PageFilter.tsx

```diff
@@ -63,6 +68,7 @@ const PageFilter = <T extends {}>({
   columns = 4,
   showResetButton,
   showSubmitButton,
+  submitOnChange,
```

Comment:
Maybe we should default it to true so we dont change the current behaviour, so if we can explicitly disable if its not required